Server/src/openbmp.cpp

```diff
@@ -417,13 +427,19 @@ void runServer(Config &cfg) {
         memcpy(cfg.c_hash_id, hash_raw, 16);
         delete[] hash_raw;
 
-        // Kafka connection
-        kafka = new msgBus_kafka(logger, &cfg, cfg.c_hash_id);
+#ifndef REDIS_ENABLED
+-        // Kafka connection
```

Comment:
>- [](http://example.com/codeflow?start=0&length=1)

build breaks.

Reply:
fixed and add sanity check for original kafka path compilation.